src/translations.js

```diff
@@ -185,7 +190,8 @@ const statCardLocales = ({ name, apostrophe }) => {
       np: "कुल योगदानहरू",
       el: "Συνεισφέρθηκε σε",
       ru: "Внёс вклад в",
-      "uk-ua": "Вніс внесок у",
+      ua: "Внески за рік",
```

Comment:
Just wondering. Why is this used instead of Google Translate's `Внесено`. This field can also show all contributions when include_all_commits` is enabled (see [include_all_commits](https://github.com/anuraghazra/github-readme-stats#stats-card-exclusive-options)?

Reply:
@BogdanDevUA, what do you think is the google translation correct or the translation which is already there? Further, do you know why there is a `uk-ua` and a UK language code (see https://github.com/ladjs/i18n-locales)? I think we can stick with `uk` since this will just be the Ukrainian language while `uk-ua` will be the ukraining language spoken in Ukraine (see https://github.com/svenfuchs/rails-i18n/issues/285). What do you think?